conda-forge.yml

```diff
@@ -25,6 +25,7 @@ os_version:
 provider:
   linux_64: github_actions
   linux_aarch64: azure
+  linux_ppc64le: default
```

Comment:
Think this can be `azure`

Though please make sure to add `linux_ppc64le` to `build_platform` earlier in this file